src/main/java/com/duke/Duke.java

```diff
@@ -0,0 +1,197 @@
+package com.duke;
+
+import com.file.FileManager;
+import com.task.Deadline;
+import com.task.Event;
+import com.task.Task;
+import com.task.Todo;
+
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.io.IOException;
+
+public class Duke {
+
+    public static FileManager fileTask;
+    static ArrayList<Task> list = new ArrayList<>();
+    public static String[] dukeLines = new String[100];
+    public static int dukeLineCount = 0;
+    public final static String LINESEPARATER =  "\t______________________________________________________________________";
```

Comment:
This constant variable has been misspelt. Also, words are conventionally separated with an underscore. Refer to Java coding standard  for [constant words](https://se-education.org/guides/conventions/java/basic.html#naming).